templates/default/services.conf.erb

```diff
@@ -115,9 +115,20 @@ filter {
     dissect {
       mapping => {"message" => "%{templogdate} %{+templogdate} %{priority} %{logger_name}: %{log_message}"}
     }
-    json {
-      source => "log_message"
-      skip_on_invalid_json => true
+		clone {
+			clones => ["onlinefs_service_log"]
+		}
```

Comment:
maybe we could instead make a clone after we know that there is `[log_arguments][project_id]`?

Reply:
It will not work. The service logs takes the log message as a string so we need to keep the original first before we convert it json.